testsuite/integration/microprofile-tck/metrics/src/test/resources/arquillian-bootable.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<arquillian xmlns="http://jboss.org/schema/arquillian" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <container qualifier="jboss" default="true" >
+        <configuration>
+            <property name="installDir">${install.dir}</property>
+            <property name="jarFile">${bootable.jar}</property>
+            <property name="jbossHome">${jboss.home}</property>
+            <property name="javaVmArguments">${microprofile.jvm.args}</property>
+            <property name="serverConfig">standalone-microprofile.xml</property>
```

Comment:
This property is not used by the bootable JAR Arquillian container adapter and it is not possible to provide a bootable JAR with a configuration file. It is built during the packaging process and is kept read only.

Reply:
Resolved with: efdbd789ff1ea6f3930e1c4584643ad9c2b6e350